models/image.py

```diff
@@ -107,7 +108,7 @@ def format(self, value):
     )
 
     ref_image_index = sa.Column(
-        sa.Integer,
+        sa.SMALLINT,
```

Comment:
I think the reason I'd rather have the image id here instead of the image index is so that we could figure out the reference using pure SQL, without having to rely on intermediate processing in python to do it.

(We probably could do it in pure SQL by sorting by MJD, but it makes the SQL more complicated and more fraught.)

(No need to change this here; you  may have already made an Issue for this.  Something to think about for the future.)